### ISSUE TITLE
[Accessibility issue] Vignettes lacks alt= in plots

### ISSUE THREAD

**Issue by mattansb** [easystats/effectsize#573]
See https://github.com/easystats/easystats/issues/352

**Comment by mattansb** [comment]
Apparently there are not plots in the vignettes!
